### ISSUE TITLE
Must work with IDDE

### ISSUE THREAD

**Issue by pdpinch** [jazkarta/obsolete-edx-sga#5]
- Instructor must be able to grant an extension through the IDDE interface
- When an extension is granted, student should be able to upload a file
- When an extension is removed, and the due date has passed, student should no longer be able to upload

**Comment by chrisrossi** [comment]
Currently the SGA itself seems to work fine with IDDE, but IDDE itself isn't working correctly.  When an extended due date is set on a homework, it doesn't cascade down the tree to individual problems, the way it should.  It may be that IDDE has experienced some bitrot with no one specifically focused on it, or it may not be interacting the XBlocks correctly.

**Comment by chrisrossi** [comment]
Indeed, what I was observing was a bug in IDDE.  I have submitted a pull request to fix this issue: https://github.com/edx/edx-platform/pull/3718

Note that an earlier pull request I submitted related to this project has so far received no feedback:
https://github.com/edx/edx-platform/pull/3615

Given that, MIT may need to make themselves a little bit of a squeaky wheel to make sure this fix goes out in a timely manner.

**Comment by pdpinch** [comment]
This still isn't working for me, but I'm assuming that's because Carson's doesn't have the IDDE fix in edx/edx-platform#3718

I'd like to leave this open until I can confirm it's fixed.

**Comment by carsongee** [comment]
I have code running now with the patch applied
